scss/typography/_helpers.scss

```diff
@@ -38,6 +38,32 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+/// If `true`, makes dropcaps uppercase.
+/// @type Boolean
+$dropcaps-uppercase: true !default;
+
+/// Dropcaps component
+@mixin dropcaps($uppercase: $dropcaps-uppercase) {
+  // Default margin (Arial/Helvetica) to let it properly aligned from the top on the 
+  // first line beside it and have a tiny space between them through the right
```

Comment:
@ncoden - If you are talking about the comment, i have used it from there only and my english isn't great. so would request you to give me the content for it and i will add ... Pleaseee ?